tests/pcap-log-lz4-05-tunnel/test.yaml

```diff
@@ -0,0 +1,14 @@
+requires:
+  features:
+    - liblz4
+  min-version: 8
+      
+pcap: ../vxlan-decoder-01/input.pcap
+
+args:
+  - --runmode=single
+
+checks:
+    - shell:
+        args: ls -l log.pcap.1525778007.671337.lz4 | wc -l | xargs
```

Comment:
Is this `log.pcap.1525778007.671337.lz4` supposed to always have the same content ?

This test would pass if the file exists and is empty for instance...

Reply:
content as in packets, yes, content as in exact bytes: no. Platform differences can make them different (e.g. endianess)

Reply:
or perhaps run tshark on it, it supports the pcap.lz4 out of the box. In any case this can be done as a follow up effort.